_docs/howto.md

```diff
@@ -162,7 +163,46 @@ Value for `search` controls inclusion of the document content into theme search
 
 Follow the sample pattern shown in the theme, specifically looking at Markdown sample documents in the `_docs` folder as an example.
 
+#### Adding Mermaid diagrams to a page
 
+Our pages now have the capability to render [Mermaid diagrams](https://mermaid.js.org/). To include a Mermaid diagram on a documentation page:
+
+1. add `mermaid: true` to the front matter of the page you want the diagram on
+2. insert the diagram following the [Mermaid syntax](https://mermaid.js.org/syntax/flowchart.html) as follows (remove backslashes as those are only needed for rendering the code here):
+
+{% raw %}
+```
+{\% raw %}
+<div class="mermaid">
+%%{init: {"flowchart": {"htmlLabels": false}} }%%
+flowchart LR
+    markdown["`This **is** _Markdown_`"]
+    newLines["`Line1
+    Line 2
+    Line 3`"]
+    markdown --> newLines
+</div>
+{\% endraw %}
+```
+{% endraw %}
```

Comment:
Try this to get it rendered as code without the need to escape the raw/endraw liquid tags.

```suggestion
<div class="language-markdown highlighter-rouge"><div class="highlight"><pre class="highlight"><code>{{ "{% raw " }}%}
&lt;div class="mermaid"&gt;
%%{init: {"flowchart": {"htmlLabels": false}} }%%
flowchart LR
    markdown["`This **is** _Markdown_`"]
    newLines["`Line1
    Line 2
    Line 3`"]
    markdown --&gt; newLines
&lt;/div&gt;
{{ "{% endraw " }}%}
</code></pre></div></div>
```

Reply:
I'm going to commit this and see what it looks like